cms/djangoapps/contentstore/views/item.py

```diff
@@ -294,6 +294,7 @@ def xblock_view_handler(request, usage_key_string, view_name):
 
         elif view_name in PREVIEW_VIEWS + container_views:
             is_pages_view = view_name == STUDENT_VIEW   # Only the "Pages" view uses student view in Studio
+            is_library_view = view_name == 'container_child_preview'
```

Comment:
Also, this method shouldn't know anything about libraries. It is unfortunate that this file already has some library-specific logic, but I would very much hope that things that are specific to libraries can remain within library code.